docs/examples/flip-a-coin.md

```diff
@@ -0,0 +1,42 @@
+# Flip A Coin
+
+A skill for opsdroid to perform flipping a coin.
+
+## Setting Up
+
+Navigating to 'opsdroid/skills/' there is a file called '__init__.py' in which will contain the functions for opsdroid to flip a coin.
```

Comment:
I'm happy if you want to add your skills inside the `opsdroid/skills` folder, but I think we should update this line a bit. An opsdroid skill is just a python function/class that is imported from somewhere.

I'd say that we could change this to something like:

"create a folder with __init__.py that we will use on opsdroid configuration." 

You could then say that perhaps using the path `opsdroid/skills` could be a good place to put your skills, but opsdroid does some file handling and will load the modules into its own directory after they are installed.